src/test/resources/net/sf/jabref/importer/fileformat/BiblioscapeImporterTestUnknownType.bib

```diff
@@ -0,0 +1,6 @@
+@misc{,
+  author = {Name},
+  booktitle = {Test Title},
+  pages = {--11},
```

Comment:
Thanks for the clarification!

I thought the test was created to "test unknown types" :wink:

But it's okay imho that you test this here, too.